docs/source/author/chapters.rst

```diff
@@ -68,9 +64,7 @@ In the following sections, each configuration file is exaplained in more detail.
 
 .. note::
 
-  - Some of the keys (:ref:`what-is-a-key`) have angle brackets around them,
-    ``<like so>``. This means that they are variables and you can call them
-    whatever you like in your configuration file (without the angle brackets).
+  - Some of the keys (:ref:`what-is-a-key`) have angle brackets around them, ``<like so>``. This means that they are variables and you can call them whatever you like in your configuration file (without the angle brackets).
 
 .. _application-structure-file:
 
```

Comment:
Okay I can't comment further down so this'll be confusing:

Perhaps put spaces in the examples appropriately for consistency, i.e
```
introduction-for-teachers:
  section-number: 1
<newline here>
further-reading:
  section-number: 2
```

`Chapter Configuration File`
- Under required filelds
  - This is where a link to the video is, again I don't know if it's required, but that's where it would be
  - Also icon is an svg nowadays, not sure how that affects things

`Chapter Sections Configuration File`
- 'Referenced in' should be `chapters/content/structure/<chapter-key>/sections/sections.yaml`